src/main/java/net/dries007/tfc/client/screen/CreateTFCWorldScreen.java

```diff
@@ -43,15 +42,23 @@ private static OptionInstance<Double> pctOption(String caption, double defaultVa
 
     private static OptionInstance<Integer> kmOption(String caption, int min, int max, int defaultValue)
     {
-        return new OptionInstance<>(caption, OptionInstance.cachedConstantTooltip(Component.translatable(caption + ".tooltip")), (text, value) -> Options.genericValueLabel(text, Component.translatable("tfc.settings.km", String.format("%.1f", value / 1000.0))), new OptionInstance.IntRange(min, max), defaultValue, value -> {});
+        return new OptionInstance<>(caption, OptionInstance.cachedConstantTooltip(Component.translatable(caption + ".tooltip")),
+            (text, value) -> Options.genericValueLabel(text, Component.translatable("tfc.settings.km", String.format("%.1f", value / 1000.0))), new OptionInstance.IntRange(min, max), defaultValue, value -> {});
+    }
+
+    private static OptionInstance<Integer> scaleOption(String caption, int min, int max, int defaultValue)
+    {
+        return new OptionInstance<>(caption, OptionInstance.cachedConstantTooltip(Component.translatable(caption + ".tooltip")),
+            (text, value) -> Options.genericValueLabel(text, Component.translatable("tfc.settings.scale", value)),
+            new OptionInstance.IntRange(min, max), defaultValue, value -> {});
     }
```

Comment:
This is a bad option, it should be represented in km, and the tooltip should be in km, log scale.

Reply:
What's the default? It sounds like scaling via 2x zooms in integer space is not your expectation of the implementation in 1.20?